libtiledbvcf/cmake/Modules/FindWinDrvLet.cmake

```diff
@@ -0,0 +1,71 @@
+set(WINDRVLET_FOUND FALSE)
```

Comment:
Please remove this file.

Reply:
Ok.  (This and several others can go when the build is no longer trying to directly perform the msys2 cmake build, as will be the case with the removal of the WIN32 section in FindHTSlib.cmake elsewhere.